src/main/java/vic/cinema/controller/MovieController.java

```diff
@@ -26,15 +26,13 @@ public MovieController(MovieService movieService, MovieMapper movieMapper) {
 
     @PostMapping
     public Movie addMovie(@RequestBody MovieRequestDto movieRequestDto) {
```

Comment:
let's make this method void and later we may return http status

Reply:
Done